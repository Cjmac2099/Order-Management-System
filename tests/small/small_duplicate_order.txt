0HWCTIHHRG
6.43
P3YC5YKC5H
82.07
WAGZ2AHLHK
40.9
78CLOCTGWB
87.67
78CLOCTGWB
47.2
P3YC5YKC5H
69.64
0J73HCH26X
56.98
END
