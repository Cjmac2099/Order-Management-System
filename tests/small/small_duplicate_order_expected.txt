Loading catalog...
Enter 'END' to finish order.
SKU> 
SKU matches product Dried Apricots, price per pound: $7.66
Quantity>
SKU> 
SKU matches product Fresh Apricots, price per pound: $3.62
Quantity>
SKU> 
SKU matches product Juice Apples frozen concentrate, price per pound: $0.61
Quantity>
SKU> 
SKU matches product Canned Apricots packed in juice, price per pound: $1.86
Quantity>
SKU> 
SKU matches product Canned Apricots packed in juice, price per pound: $1.86
Quantity>
SKU> 
SKU matches product Fresh Apricots, price per pound: $3.62
Quantity>
SKU> 
SKU matches product Fresh Apples, price per pound: $1.85
Quantity>
SKU> 
Printing order summary:
6.43lbs Dried Apricots: $49.26
82.07lbs Fresh Apricots: $296.78
40.90lbs Juice Apples frozen concentrate: $24.89
87.67lbs Canned Apricots packed in juice: $163.46
47.20lbs Canned Apricots packed in juice: $88.00
69.64lbs Fresh Apricots: $251.83
56.98lbs Fresh Apples: $105.65
